MapboxNavigationTests/InstructionsBannerViewSnapshotTests.swift

```diff
@@ -5,181 +5,181 @@ import MapboxDirections
 @testable import MapboxCoreNavigation
 
 class InstructionsBannerViewSnapshotTests: FBSnapshotTestCase {
-    
```

Comment:
Why does this keep happening? Someone's got weird Xcode settings.

Reply:
This isn’t a great venue to discuss whitespace settings, but for what it’s worth, I’m not as fond of unindenting blank lines. It tends to result in less intuitive merge conflicts by default. In any case, both GitHub and Xcode have options for hiding whitespace changes from the diff.